common/src/option.c

```diff
@@ -63,11 +63,19 @@ static bool int_parser(struct Option * opt, const char * str)
 
 static bool bool_parser(struct Option * opt, const char * str)
 {
+  //rtrim
+  char * val = strdup(str);
+  size_t len = strlen(val);
+  while (len > 1 && isspace(val[len-1]))
+    --len;
+  val[len] = '\0';
```

Comment:
Can we just use a stack buffer with 5 characters, then fails for longer strings and then `memcpy`?

Reply:
How will that detect "1_" (with "_" being space)?

Reply:
You use the same logic here to compute the "actual" length.